src/odb/include/odb/geom.h

```diff
@@ -176,6 +177,9 @@ class Oct
   int yMax() const;
   std::vector<Point> getPoints() const;
 
+  Oct bloat(int margin) const;
+  Rect getRect() const;
```

Comment:
I think `getEnclosingRect` is clearer as the rect is not equivalent to the oct.

Reply:
Probably makes sense to correct name in Polygon as well